### PR TITLE
Neyyi karam dosa

### PR DESCRIPTION
**Pull request by guru-sulakhe**

### CODE REVIEW

File: menu.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 dosa 
-karam
-oil
\ No newline at end of file
+extra karam
```
Comment (guru-user) [inline menu.txt:2]: do extra karam is required?

Comment (guru-sulakhe) [reply menu.txt:2]: yes it is required.

Comment (guru-user) [reply menu.txt:2]: yeah that is fine

**Review by guru-user — APPROVED** [review]
neyyi karam dosa is good

**Pull request merged**